src/helix-ui/elements/HXErrorElement.less

```diff
@@ -1,2 +1,6 @@
 @import './HXElement';
 @import 'elements/hx-icon';
+
+#icon {
+  margin-right: 0.25rem;
+}
```

Comment:
Given that we no longer have misinterpreted whitespace, we now have to apply a margin to adhere to redline specs.